src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Draw, FigureArea {
```

Comment:
```suggestion
public class Circle extends Figure {
```

все наследники класса Figure автоматически имплементируют все интерфейсы, которые импелементирует родительский класс